haskell-style.md

```diff
@@ -41,6 +41,18 @@ done for a few reasons:
 - This gives us a chance to see just how much time we've saved by automating
   what used to be a (possibly tenuous) negotiated consensus
 
+## Whitespace
+
+- Prefer two space indentation
+
+  Exceptions include anything automated (e.g. de-dented `where`) and the leading
+  tabs required in `Makefile`s.
```

Comment:
Do we have a method of automating the formatting of make files? The tabs+spaces in make files is very wacky and without automation it is just an invitation for error.

Reply:
> Do we have a method of automating the formatting of make files? The tabs+spaces in make files is very wacky

My editor (vim) just does it, so yes, but I don't know what's driving it -- I don't have any plugin specifically for that. Yours doesn't?

It's possible it's an Arch system configuration? Every `Makefile` I've ever come across has worked that way, but I mostly encounter `Makefile`s in an Arch or Arch-adjacent ecosystem.